src/content/configuration/module.mdx

```diff
@@ -49,6 +49,27 @@ module.exports = {
 };
 ```
 
+## `module.unsafeCache`
+
+`boolean` `function ()`
+
+Cache the resolution of module requests. There are couple of defaults for `module.unsafeCache`:
+
+- `false` if [`cache`](/configuration/other-options/#cache) is disabled.
+- `true` if [`cache`](/configuration/other-options/#cache) is enabled and the module appears to come from node_modules otherwise `false`.
```

Comment:
Nit: do you mind changing this to:
> ...come from node modules,  **`false` otherwise.**

Reply:
```suggestion
- `true` if [`cache`](/configuration/other-options/#cache) is enabled and the module appears to come from node modules, `false` otherwise.
```